deploy/kustomize/base/deployment.yaml

```
@@ -22,6 +22,7 @@ spec:
             - --health-probe-bind-address=:8081
             - --metrics-bind-address=0.0.0.0:9090
             - --leader-elect
+#            - --max-concurrent-reconciles=1
```

I did not know we had formatting requirements for yaml comments 😅 
```suggestion
            # - --max-concurrent-reconciles=1
```